Backend/Remora.Discord.Rest/API/Emoji/DiscordRestEmojiAPI.cs

```diff
@@ -58,6 +58,115 @@ ICacheProvider rateLimitCache
     {
     }
 
+    /// <inheritdoc />
+    public Task<Result<IReadOnlyList<IEmoji>>> ListApplicationEmojisAync
+    (
+        Snowflake applicationID,
+        CancellationToken ct = default
+    )
+    {
+        return this.RestHttpClient.GetAsync<IReadOnlyList<IEmoji>>
+        (
+            $"applications/{applicationID}/emojis",
+            "items",
```

Comment:
This is by no means your fault - it's Discord's, somewhat, but this fails with an exception when the app doesn't have any emojis.

`The input does not contain any JSON tokens. Expected the input to start with a valid JSON token, when isFinalBlock is true.`

The returned JSON is in fact, correct: `{"items":[]}`, but STJ just...*dies*.